src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.md

```diff
@@ -54,7 +39,16 @@ return (
         schemaId={schemaId}
         liveValidate={liveValidate}
         onSuccess={() => {
-          setShowToast(true)
+          displayToast(
```

Comment:
This is really convenient.  We all need to start using this!

Reply:
The signature is pretty ugly (since it's probably mirroring the signature of the old DisplayUtils in the SWC project).  I wonder if you can add another displayToast() version that takes in an object (where many parameters are optional) to avoid all of these nulls.  What do you think?

Reply:
Definitely. The only necessary params are the message and maybe the variant. I'd be ok with changing the signature to something like
```ts
type ToastOptions = {
	variant?: 'info' | 'success' | 'warning' | 'danger' // default 'info'
	title?: string // default undefined
	autoCloseInMs?: number // default 15000
	primaryButtonText?: string // default undefined
	onPrimaryButtonClick?: () => void // default undefined
	secondaryButtonText?: string // default undefined
	secondaryButtonHref?: string // default undefined
}

function displayToast(message: string, opts: SynapseToastOptions);
```